### ISSUE TITLE
Add Python Templates to the SDK generator

### ISSUE THREAD

**Issue by eldadfux** [appwrite/sdk-generator#6]
We need help adding Python templates to this project to allow easy generation of Python SDKs for the Appwrite server. We already have some templates ready for Python but we still need to test them and complete the SDK features checklist to provide full support for developers.

A full guide on how to add new language support to the Appwrite SDK generator is located here:
https://github.com/appwrite/sdk-generator/blob/master/CONTRIBUTING.md

**Comment by udaykiranreddykondreddy** [comment]
Hello @eldadfux Can I work on this?

**Comment by eldadfux** [comment]
Hey @udaykiranreddykondreddy that will be great! Please reach out to me on Gitter for any question!

I am assigning you to this issue.

**Comment by kervi** [comment]
What is the status for Python support?
I have looked into the generated python project and have these comments to the templates.

in setup.py.twig the package line should be
packages = ['{{spec.title | caseSnake}}', '{{spec.title | caseSnake}}/services'],

I dont know the twig syntax but the result for services py files should be something like this in the constructor.
```
    class Account(Service):
        def __init__(self, client):
            super(Account, self).__init__(client)
```

**Comment by eldadfux** [comment]
@kervi the python support is still a work in progress. It could be really helpful if you would code-review this very slim version of SDK generated by the latest version of the python twig templates:
https://github.com/eldadfux/python-sdk-sample

This way I will be able to very easily translate your feedback to the TWIG  templates syntax and get closer to a working Appwrite python SDK.

**Comment by kervi** [comment]
A service class should start like this with the constructor:

```
class Foo(Service):
    def __init__(self, client):
        super(Foo, self).__init(client)

    ... get and other methods

```

**Comment by kervi** [comment]
In setup.py the packages field should look like this

setup(
  name = 'appwrite',
  packages = ['appwrite', 'appwrite/services'],
  .... rest of setup

**Comment by kervi** [comment]
The download url is spooky

download_url='https://github.com///archive/.tar.gz',

I guess that it should look like this
download_url='https://github.com/appwrite/sdk-for-python/[path to tar ball]',

As you have not made any releases there are any files to link to.

**Comment by eldadfux** [comment]
@kervi I have updated the templates according to your feedback. This is demo SDK:
https://github.com/eldadfux/python-sdk-sample looking better?

**Comment by kervi** [comment]
Looks right




---- On lø, 16 nov. 2019 19:37:20 +0000 notifications@github.com wrote ----



@kervi I have updated the templates according to your feedback. This is demo SDK:
https://github.com/eldadfux/python-sdk-sample looking better?

**Comment by PedroCisnerosSantana** [comment]
not working with the past setup and two classifiers are wrong
```
-from distutils.core import setup
+import setuptools

-setup(
+setuptools.setup(

-'Development Status :: 5 - PRODUCTION/STABLE',
+'Development Status :: 5 - Production/Stable',

-'License :: OSI Approved :: BSD-3-Clause',
+'License :: OSI Approved :: BSD License',
```

**Comment by eldadfux** [comment]
Just released version 0.0.6 of the Python SDK which now support file uploading and include some other bug fixes. Next version will probably be 1.0
